examples/spark/spark-gluster/spark-master-service.yaml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
I've used all the spark-gluster example yamls (with the values tweaked for my cluster) with kubectl -f and they worked as expected.  I'm new to yaml; to generate the files, I used a json to yaml converter.